Update Guidelines Incipit and Explicit

CarstenHoffmannMarburg opened the pull request:
I updated the guidelines concerning the incipits and explicits as discussed recently in our latest webmeeting. I am not sure, whether the phrase on colophons is useful or should deleted or extended. It might be appropriate to refer to the subpage "colophon", but I did not know, how I can indicate it in a `<ref>`.

DenisNosnitsin1970 commented on elements/explicit.xml:
```diff
@@ -25,7 +25,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          <div>
             <head>explicit</head>
-            <p/>
+            <p>Explicits are meant to give the user an impression of the texts included and help to navigate in a manuscript. They should 
```
Some remarks. The wording "to give an impression" does not match the style of the guidelines. Quoting incipits and explicits serves the purpose of the description of the manuscript's content, and, specifically,  it is supposed 1) to help in identifying the texts or their parts; 2) to help in identifying the extention and completness of the texts or their parts in the manuscript.  I would say "Quotation should **preferebly** not include "common formulas or eulogies". I would not completely exclude the possibility for keeping formulaic expressions, as in some cases quoting initial formulas may be of interest or of importance (in older manuscripts they are different from the more recent). Also, sometimes the name of the commissioner can be mentioned in the formulaic part, and it may be better to encode a small piece of it, to keep te name.  I would also not say anything about later additions, since encoding added passages (if it somehow extends the original text?) as such is possible, the word "addition" can be interpreted in various ways.  
Another thing - if the text starts / ends _in abrupto_, we still quote the first words and call it "incipit" and "explicit", right? Should we mention these cases? (then quotation will be a little bit mechanical).
What I always do as cataloguer, I encode rubrications, but I don't know if we should ask for this. 
It is good to distinguish the point of view of the cataloguer (what I indicated before the point) and the pure user. But please check that the verb "to navigate" can be used in the way you do, I think "one navigates smth". Another thing - we speak about "explicit" (and better to keep it), does it interfere with "desinit"?

CarstenHoffmannMarburg replied:
>Some remarks. The wording "to give an impression" does not match the style of the guidelines. Quoting incipits and explicits serves the purpose of the description of the manuscript's content, and, specifically, it is supposed 1) to help in identifying the texts or their parts; 2) to help in identifying the extention and completness of the texts or their parts in the manuscript. I would say "Quotation should preferebly not include "common formulas or eulogies". I would not completely exclude the possibility for keeping formulaic expressions, as in some cases quoting initial formulas may be of interest or of importance (in older manuscripts they are different from the more recent). Also, sometimes the name of the commissioner can be mentioned in the formulaic part, and it may be better to encode a small piece of it, to keep te name. I would also not say anything about later additions, since encoding added passages (if it somehow extends the original text?) as such is possible, the word "addition" can be interpreted in various ways.

I tried a rework, that will hopefully meet all your quests. I also had some problems with the idea to avoid any kind of reliogious formulas.

CarstenHoffmannMarburg replied:
>What I always do as cataloguer, I encode rubrications, but I don't know if we should ask for this.

I agree, but I would not recommend to comment on that when describing incipit and explicit in a general way.

CarstenHoffmannMarburg replied:
>It is good to distinguish the point of view of the cataloguer (what I indicated before the point) and the pure user. But please check that the verb "to navigate" can be used in the way you do, I think "one navigates smth". Another thing - we speak about "explicit" (and better to keep it), does it interfere with "desinit"?

Personally, I used these terms interchangeably and I think their is no consensus about them, since both mean "end of a text" it may mean the end of the writing in a specific witness or the end of a work or the end of a planned writing process. I used the term desinit in my dissertation to describe the ever extended end of the text. For BM, I needed to change my terminology to "explicit". I am not aware, that the word desinit is used in BM/TEI in any context.

thea-m commented:
I would say "20-30 words belonging to the text proper, excluding religious formulas and supplications". I would also generally encode religious formulas and certainly supplications if they include information about persons involved in the manuscript description, but it is important for cataloguers to understand that these do usually not contribute much in identifying the text.

thea-m reviewed (changes requested):
Thank you! I think this is a real improvement to the guidelines.
I disagree with the formulation "should preferably not include common formulas or eulogies unless there is no special reason to do so". I think it might be understood that it is not desirable to encode common formulas or eulogies. this is not the case, but the proper incipits/explicits should not be counted only as those that quote the beginning or end of the text proper. Any additional formulas, supplications etc. can also be encoded.

DenisNosnitsin1970 commented:
The reason not to encode formulas stems from the definition of the incipit/explicit, and from the necessity to avoid the practice when some encode only formulas and stop where the work proper starts, thinking that the incipit is already recorded. There are many cases when the formulas and eulogies have to be dropped for the sake of the proper beginning of the work. We ourselves do that, even more if we want to reduce some limitation on the quoted text. What I say is more or less what Dorothea says, perhaps we need to think a little more on the wording, to express the idea in the clearest possible way.

eu-genia commented on elements/explicit.xml:
```diff
@@ -25,7 +25,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
          <div>
             <head>explicit</head>
-            <p/>
+            <p>Quoting incipits serves the purpose of the description of the manuscript's content e.g. to help to identify the texts or their parts and 
```
this abstract should be about explicits

CarstenHoffmannMarburg commented:
I think, I solved all requests and remarks by @DenisNosnitsin1970 and @thea-m. Can this branch be approved and merged?

DenisNosnitsin1970 reviewed (changes requested):
This is ok, I only wonder if "e.g." fits here well. It means "for example", perhaps "i.e." "that is..." would work out better (for both the use of comma before and after seems to be recommendable).

CarstenHoffmannMarburg commented:
> This is ok, I only wonder if "e.g." fits here well. It means "for example", perhaps "i.e." "that is..." would work out better (for both the use of comma before and after seems to be recommendable).

OK. I changed that and rewrote some other phrases to make it a bit smoother.

nafisa-valieva commented:
@CarstenHoffmannMarburg Hi! I came across this issue and think it would be beneficial to merge your updates!
